src/components/ModelingSidebar/ModelingPanes/MemoryPane.tsx

```diff
@@ -111,7 +111,7 @@ export const processMemory = (programMemory: ProgramMemory) => {
         ?.map?.(({ identifier }: any) => identifier?.name || '')
         .join(', ')})__`
     } else {
-      processedMemory[key] = val.value
```

Comment:
@jtran @franknoirot Hey, on this line `val` is now the type `never`. So I'm not sure what to do here, it basically seems to be saying that this final `else` branch will never actually be triggered. Any suggestions for what to do here? Should I just `return processedMemory`?